src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -8077,5 +8077,403 @@ .maxstack  2
 }
 ");
         }
+
+        [Fact]
+        public void ForeachStatement_Reorder()
```

Comment:
Actually... let's make the ArrayOfTuples test more interesting and make the type of the variable something like an array of tuples of anonymous types of a List<tuple>. You can also add more variables of more complex types, including pointers, ref variables, etc.

And rename the test to ComplexTypes, or something like that.

 

---
In reply to: [115574092](https://github.com/dotnet/roslyn/pull/19283#discussion_r115574092) [](ancestors = 115574092,115573899)